File: xsolla-store-sdk/src/main/java/com/xsolla/android/store/entity/response/common/Group.java
package com.xsolla.android.store.entity.response.common;

import com.google.gson.annotations.SerializedName;

public class Group {

    @SerializedName("external_id")
    private String externalId;
    private String name;

    public String getExternalId() {
        return externalId;
    }

    public String getName() {
        return name;
    }
}

File: xsolla-store-sdk/src/main/java/com/xsolla/android/store/entity/response/common/Group.kt
package com.xsolla.android.store.entity.response.common

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class Group(
    @SerializedName("external_id")
    val externalId: String? = null,
    val name: String? = null
) : Parcelable